exasol_sphinx_github_pages_generator/deploy_github_pages.py

```diff
@@ -58,18 +75,5 @@ def deploy_github_pages(argv):
             deployer.clean_worktree(original_workdir)
 
 
-_SUCCESS = 0
-_FAILURE = 1
-
-
-def main():
-    try:
-        deploy_github_pages(sys.argv[1:])
-        return _SUCCESS
-    except Exception as ex:
-        Console.stderr(ex)
-        return _FAILURE
-
-
 if __name__ == "__main__":
-    sys.exit(main())
+    main()
```

Comment:
does click automatically report a negative exit code if it fails?

Reply:
I thought so, but now when you ask, I am not sure, anymore. @umitbuyuksahin can you check this

Reply:
I checked, click takes care of the basic error code (for common cases e.g. strg + c). Still we have a change in behaviour here, which is not 100% ideal. If an exception is risen it will crash "hard" and print the stack trace to the user. Compared to the previous behaviour where an error message was printed and a negative return value was reported.